cfe_internal/enterprise/federation/federation.cf

```diff
@@ -202,6 +202,8 @@ bundle agent transport_user
                  not( fileexists( "$(ssh_priv_key)" )));
 
     selinux_enabled.missing_ssh_context::
+      "semanage fcontext -a -t ssh_home_t $(home)/.ssh/"
+        contain => default:in_shell;
```

Comment:
AFAICT, this will run **after** *ssh-keygen* above.

Reply:
I will move it before since order inside of a promise type is based on occurrence in the file right? Otherwise multiple passes would take care of it.

Reply:
Now I'm not sure. I tested again and it seems ssh-keygen works fine. `semanage` is an add-on package we would have to install with `yum -y install policycoreutils-python` which doesn't seem reasonable at all. I'll test again on centos6 with the latest nightly and possibly close this PR. :+1: